src/Illuminate/Queue/BeanstalkdQueue.php

```diff
@@ -148,7 +148,7 @@ public function deleteMessage($queue, $id)
      */
     public function getQueue($queue)
     {
-        return $queue ?: $this->default;
+        return $this->getQueuePrefix().($queue ?: $this->default);
```

Comment:
I meant a little different. I mean create new method like `resolveQueueName($queue)` and call it in `getQueue`.
```php
protected function resolveQueueName($queue) : string
{
  return $this->getQueuePrefix() . ($queue ?? $this->default);
}
```